core/src/test/java/org/mskcc/cbio/portal/util/TestCancerStudyReader.java

```diff
@@ -68,8 +68,25 @@ public void testCancerStudyReader() throws Exception {
          cancerStudy = CancerStudyReader.loadCancerStudy( file );
          fail( "Should have thrown DaoException." );
       } catch (Exception e) {
-         assertTrue( e.getMessage().equals( 
-                  "brcaxxx is not a supported cancer type."));
+    	 assertEquals("brcaxxx is not a supported cancer type.", e.getMessage());
       }
    }
+
+	@Test
+	   public void testCancerStudyReaderShortName() throws Exception {
+
+	      File file = new File("target/test-classes/cancer_study.txt");
```

Comment:
@ersinciftci seems to to this in `src/test/resources/` instead of `target/test-classes/`. I believe `src/test/resources/` avoids a compilation step when just changing the test file, so I prefer that.

see https://github.com/cBioPortal/cbioportal/pull/963/files#r58565966